scripts/jenkins/push_docker.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+# This script is present on workers but may not be present in a development
+# environment.
+
+if [ ${WORKSPACE+x} ]  # We are on a CI worker
+then
+  source /usr/local/bin/bash_standard_lib.sh
+fi
+
+readonly RETRIES=3
+
+# This script is intended to work in conjunction with the build_docker
+# script. It assumes that build_docker.sh has been run at least once, thereby
+# creating a Docker image to push. If this script does not detect an image
+# to be uploaded, it will fail.
+
+# This script is intended to be run from a CI job and will not work if run in
+# standalone manner unless certain envrionment variables are set.
+
+# 1. Grab the tag we are working with
+
+#FIXME DRY between Docker scripts
+echo "INFO: Fetching latest tag"
+#TODO Move this into apm-pipeline-library
+CUR_TAG_DEFAULT=$(git describe --abbrev=0|sed s/^v//)
```

Comment:
Should it use the same approach as the one for building?

Reply:
Fixed.